apps/client/src/widgets/community/components/user-comment-info/user-comment-info.tsx

```diff
@@ -0,0 +1,60 @@
+import { Avatar, TextButton } from '@bds/ui';
+
+import { CommentType } from '@widgets/community/types/community-comment.type.ts';
+
+import { Image } from '@shared/types/type.ts';
+
+import * as styles from './user-comment-info.css';
+
+const DELETE_CONTENT = '삭제';
+
+interface UserCommentInfoProps {
+  comment: CommentType;
+  imageUrl: Image[];
+}
+
+const UserCommentInfo = ({ comment, imageUrl }: UserCommentInfoProps) => {
+  const {
+    content,
+    writerNickName,
+    createdAt,
+    profileImage,
+    isCommentOwner,
+    onClickDelete,
+  } = comment;
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.commentContainer}>
+        <div className={styles.userInfoContainer}>
+          <div className={styles.userInfo}>
+            <Avatar size="md" src={profileImage} />
+            <div>
+              <h2 className={styles.nickName}>{writerNickName}</h2>
+              <p className={styles.timestamp}>{createdAt}</p>
+            </div>
+          </div>
+          <div className={styles.button}>
+            {isCommentOwner ? (
+              <TextButton color="black" onClick={onClickDelete} size="sm">
+                {DELETE_CONTENT}
+              </TextButton>
+            ) : (
+              ''
+            )}
```

Comment:
```suggestion
           {isCommentOwner && (
              <TextButton color="black" onClick={onClickDelete} size="sm">
                {DELETE_CONTENT}
              </TextButton>
            )}
```

삼항연산자를 쓸 필요가 없을 것 같아요 ! 이런식으로 수정하는 건 어떨까요 ??

Reply:
완전 맞는말 👍👍